### ISSUE TITLE
Error with Ember 2.13.1

### ISSUE THREAD

**Issue by NullVoxPopuli** [sweettooth/ember-cli-stripe#28]
This is the error I see in the chrome console:
```
loader.js:232 Uncaught Error: Could not find module ember-invoke-action imported from ember-cli-stripe/services/stripe
    at missingModule (loader.js:232)
    at findModule (loader.js:243)
    at Module.findDeps (loader.js:173)
    at findModule (loader.js:247)
    at Module.findDeps (loader.js:173)
    at findModule (loader.js:247)
    at requireModule (loader.js:26)
    at resolveInitializer (index.js:29)
    at registerInitializers (index.js:42)
    at exports.default (index.js:24)
﻿```
```

The packages in the node_modules/ember-cli-stripe/node_modules directory

```
babel-core                 core-js          home-or-tmp  minimatch
babylon                    detect-indent    json5        path-exists
bluebird                   ember-cli-babel  js-tokens    source-map-support
broccoli-babel-transpiler  globals          lodash       user-home
```

the package.json file says version 1.0.0 (which I believe is current).

there is a _dev_Dependency:
    "ember-invoke-action": "1.4.0",

but it is not installed when I do yarn install or npm install.

​

**Comment by ryanlitalien** [comment]
@NullVoxPopuli I also had the same problem.
```
ember.debug.js:6008 DEBUG: Ember              : 2.12.1
ember.debug.js:6008 DEBUG: Ember Data         : 2.12.2
```
So I did `yarn add ember-invoke-action@1.4.0`, which added this to my package.json:

```
  "dependencies": {
    "ember-invoke-action": "1.4.0"
  }
```
And the error has disappeared. Hopefully that'll help for the time being.

**Comment by vladucu** [comment]
thx for the report!
will try to check this sometime this week, but for now the above should be a work-around

**Comment by vladucu** [comment]
this should be fixed in `1.0.1`...let me know if you have any problems with this version
